ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/longrun/openshift-e2e-test-hypershift-qe-mgmt-longrun-chain.yaml

```diff
@@ -5,5 +5,18 @@ chain:
   - ref: idp-htpasswd
   - ref: openshift-extended-test-longduration
   - ref: openshift-e2e-test-qe-report
+  env:
+    - name: TEST_SCENARIOS
```

Comment:
We seldomly override this var, so it's better not to override it here.

Reply:
Have to override them, because the original openshift-extended-test-longduration step missed some components, some cases of them won't be executed in longduration jobs.
```
$ cat ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-ref.yaml
ref:
  as: openshift-extended-test-longduration
  from: tests-private
  grace_period: 10m
  commands: openshift-extended-test-longduration-commands.sh
  cli: latest
  timeout: 22h0m0s
  env:
  - name: TEST_SCENARIOS
    default: |
        ETCD|PSAP|SDN|STORAGE|Authentication
        OLM|Network_Edge|Operator_SDK
        Image_Registry|Container_Engine_Tools
        NODE|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING|CFE
```

Reply:
We should add those needed but missing one in it (rather than override it).  You can see those two lists have been diverged, that's something we should avoid.

Reply:
BTW, so do you mean for hypershift mgmt test jobs you expected that api/workload total cases will not grow somehow? Or when it grows more than the maximum 24 hours, we need to divide them again? In addition, I guess since the `.*Critical` make the total case number a bit less, is this expected?